tests/neo.UnitTests/SmartContract/UT_Helper.cs

```diff
@@ -27,5 +30,105 @@ public void TestIsMultiSigContract()
             };
             Assert.IsFalse(case2.IsMultiSigContract());
         }
+
+        [TestMethod]
+        public void TestIsSolidTransfer()
+        {
+            // PUSHX
+
+            using (ScriptBuilder sb = new ScriptBuilder())
+            {
+                sb.EmitPush(12);
+                sb.EmitAppCall(UInt160.Zero, "balanceOf", UInt160.Parse("01ff00ff00ff00ff00ff00ff00ff00ff00ff00a4"));
+                sb.Emit(OpCode.EQUAL);
+                sb.Emit(OpCode.ASSERT);
+
+                Assert.IsTrue(sb.ToArray().IsSolidTransfer());
+            }
+
+            // PUSHM1
+
+            using (ScriptBuilder sb = new ScriptBuilder())
+            {
+                sb.EmitPush(-1);
+                sb.EmitAppCall(UInt160.Zero, "balanceOf", UInt160.Parse("01ff00ff00ff00ff00ff00ff00ff00ff00ff00a4"));
```

Comment:
Do we have some test where Balance was not Zero before? I mean, guy has 10 tokens, is sending 6... thus, balance should be 4 (in case of OpCode.EQUAL); and assertion should be ">=6", if using OpCode.GEQ.
Let's call these two strategies "strict" (for OpCode.EQUAL) and "flexible" (for OpCode.GEQ).